index.html

```diff
@@ -21,23 +22,191 @@
+                        <span>2015</span>
+                    </div>
+                    <p>A daily selection of privately personalized reads; no accountsor or sign-ups required.</p>
+                    <ul>
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javaScript</li>
+                    </ul>
+                    <button class="bb">See Project</button>
+                </div>
+            </div>
+
+            <!-- third card -->
+            <div class="work-card">
+                <img src="/image2/Snapshoot Portfolio 3.svg" alt="snapshoot Portfolio" class="img-project">
+                <div class="left-block">
+                    <h2>Tonic</h2>
+                    <div class="frame">
+                        <span class="CANO">CANOPY</span>
+                        <img src="/image2/Counter.svg" alt="circul icon">
+                        <span>Back End Dev</span>
+                        <img src="/image2/Counter.svg" alt="circul icon">
+                        <span>2015</span>
+                    </div>
+                    <p>A daily selection of privately personalized reads; no accountsor or sign-ups required.</p>
+                    <ul>
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javaScript</li>
+                    </ul>
+                    <button class="bb">See Project</button>
+                </div>
+            </div>
+
+            <!-- fourth card -->
+            <div class="work-card">
+                <img src="/image2/Snapshoot Portfolio 4.svg" alt="snapshoot Portfolio" class="img-project">
+                <div class="left-block">
+                    <h2>Tonic</h2>
+                    <div class="frame">
+                        <span class="CANO">CANOPY</span>
+                        <img src="/image2/Counter.svg" alt="circul icon">
+                        <span>Back End Dev</span>
+                        <img src="/image2/Counter.svg" alt="circul icon">
+                        <span>2015</span>
+                    </div>
+                    <p>A daily selection of privately personalized reads; no accountsor or sign-ups required.</p>
+                    <ul>
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javaScript</li>
+                    </ul>
+                    <button class="bb">See Project</button>
+                </div>
+            </div>
+        </section>
+
+        <!-- about my self section -->
+        <section  id="sec3">
+            <!-- Top frame -->
+            <h2>About <br> Myself</h2>
+            <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and
+                experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <a href="#img-svg" id="connect">LET’S CONNECT</a>
```

Comment:
- [OPTIONAL]  Great job 💪 on implementing many project requirements! Please replace the `a` tag with `hx` tag (such as h2 or h3)  since this is a title. This is considered the best practice in HTML semantics.